src/main/java/Main.java

```diff
@@ -0,0 +1,110 @@
+            } else if (msg.startsWith("mark ")) {
+                int index = Integer.parseInt(msg.substring(5));
+                list[index - 1].mark();
+                System.out.println(
+                        NL + DIVIDER + NL +
+                                "Congratulations! The following task has been marked as complete:" +
+                                NL + "  " + list[index - 1] + NL +
+                                DIVIDER + NL
+                );
+            } else if (msg.startsWith("unmark ")) {
+                int index = Integer.parseInt(msg.substring(7));
+                list[index - 1].unmark();
+                System.out.println(
+                        NL + DIVIDER + NL +
+                                "Got it! The following task has been marked as incomplete:" +
+                                NL + "  " + list[index - 1] + NL +
+                                DIVIDER + NL
+                );
+            } else {
+                int index;
+
+                if (msg.startsWith("todo ")) {
+                    index = 5;
+                    if (msg.substring(index).isBlank()) {
+                        System.out.println(
+                                NL + DIVIDER + NL +
+                                        "You may forget things if the description was left blank! Try adding a message!" +
+                                        NL + DIVIDER + NL
+                        );
+                        repeat = true;
+                        continue;
+                    }
+                    list[count] = new Todo(msg.substring(index));
+                } else if (msg.startsWith("deadline ")) {
+                    index = 9;
+                    int indexDdl = msg.indexOf("ddl:");
+                    list[count] = new Deadline(msg.substring(index, indexDdl), msg.substring(indexDdl + 5));
+                } else if (msg.startsWith("event ")) {
+                    index = 6;
+                    list[count] = new Event(msg.substring(index), "", "");
+                } else {
+                    repeat = true;
+                    System.out.println(
+                            NL + DIVIDER + NL +
+                                    "The supported types and corresponding formats are as follows:" + NL +
+                                    "To-Dos: todo [description]" + NL +
+                                    "Deadlines: deadline [description] ddl: [DD-MM-YY HH:MM]" + NL +
+                                    "Events: event [description] from: [DD-MM-YY HH:MM] to: [DD-MM-YY HH:MM]" + NL +
+                                    NL + "Examples:" + NL +
+                                    "todo Do the laundry" + NL +
+                                    "deadline CS2113 Increment L5 ddl: 14-02-25 16:00" + NL +
+                                    "event Josh's birthday party from: 03-03-25 15:00 to: 04-03-25 00:00" +
+                                    NL + DIVIDER + NL
```

Comment:
Since the print statements are common for many commands -  you can consider creating a function instead of repeating the same code lines for each command